### PR TITLE
✨ New demo - File System Access API

### PR DESCRIPTION
**Pull request by max-programming**

### CODE REVIEW

**Comment by max-programming** [comment]
Don't merge yet. Fixing a problem

**Comment by atapas** [comment]
@max-programming, Please let me me know when it is ready to merge

**Comment by max-programming** [comment]
> @max-programming, Please let me know when it is ready to merge

I think it's the right time. Do you want me to add something more?

**Comment by atapas** [comment]
> > @max-programming, Please let me know when it is ready to merge
> 
> @atapas I think it's the right time. Do you want me to add something more?

Would u update the Readme also adding this as a new inclusion? @max-programming

**Comment by max-programming** [comment]
> > > @max-programming, Please let me know when it is ready to merge
> > 
> > @atapas I think it's the right time. Do you want me to add something more?
> 
> Would u update the Readme also adding this as a new inclusion? @max-programming 

Yea sure

**Review by atapas — CHANGES_REQUESTED** [review]
@max-programming please update the readme with this new inclusion.

File: README.MD [hunk 1]
```diff
@@ -31,9 +31,10 @@ The Demo is accessible from here: [http://js-webapis.netlify.app/](http://js-web
 Build Staus: [![Netlify Status](https://api.netlify.com/api/v1/badges/2c518303-b5c8-4be8-b1a3-2b73affb7e9b/deploy-status)](https://app.netlify.com/sites/js-webapis/deploys)
 
 So far, it includes the following examples:
+1. 📂 File System API
 1. 📺 FullScreen API
-1. 📋 Clipboard API
-1. 📡 Broardcast
+1. 📋 Broadcast API
```
Comment (atapas) [inline README.MD:36]: @max-programming, I think there is a small mistake. The Clipboard got renamed to Broadcast by mistake

image

**Comment by atapas** [comment]
@all-contributors please add @max-programming  for code

**Comment by max-programming** [comment]
@atapas maybe because of the uppercase MD

**Comment by atapas** [comment]
> @atapas maybe because of the uppercase MD 

Yes, I also thought so. Once you take care of this comment(https://github.com/atapas/webapis-playground/pull/3#discussion_r654888810), I shall merge this PR. After that, I will make it `Readme.md` and initiate the all-contributor once again. Let me know when ur changes are done.

File: README.MD (hunk 1 above)
Comment (max-programming) [reply README.MD]: Done! Thanks for pointing that out

**Review by atapas — APPROVED** [review]
Tested. Worked like a charm. Hence approving.

**Pull request merged**